Libraries/Text/TextInput/Singleline/RCTUITextField.m

```diff
@@ -108,6 +108,7 @@ - (instancetype)initWithFrame:(CGRect)frame
 #if TARGET_OS_OSX // [TODO(macOS ISS#2323203)
     self.bordered = NO;
     self.accessibilityRole = NSAccessibilityTextFieldRole;
+    self.backgroundColor = [NSColor clearColor];
```

Comment:
Does RCTUITextView.m for multi-line TextInputs just work when `backgroundColor` is set?

Reply:
I think so?
<img width="752" alt="image" src="https://user-images.githubusercontent.com/4123478/84488958-407f0600-aca1-11ea-9408-f4a4d231b297.png">
```diff
diff --git a/RNTester/js/examples/TextInput/TextInputExample.ios.js b/RNTester/js/examples/TextInput/TextInputExample.ios.js
index f21ca72d3..c8bb90377 100644
--- a/RNTester/js/examples/TextInput/TextInputExample.ios.js
+++ b/RNTester/js/examples/TextInput/TextInputExample.ios.js
@@ -530,6 +530,7 @@ const styles = StyleSheet.create({
     padding: 4,
   },
   multiline: {
+    backgroundColor: '#ff00ff',
     borderWidth: StyleSheet.hairlineWidth,
     borderColor: '#0f0f0f',
     flex: 1,
```